### PR TITLE
[@types/coingecko-api] Added type definition for `CoinsFetchData.tickers`

### PR DESCRIPTION
**Pull request by JonathanTheZero**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://www.coingecko.com/de/api/documentation> (Tickers is mentioned as a property, the previous type was just `[]`, which was compiled to `never[]`, I can provide example code for using the added interfaces, if requested)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/coingecko-api/index.d.ts [hunk 1]
```diff
@@ -313,7 +314,43 @@ interface CoinsFetchData {
...
+    timestamp: Date;
+    last_traded_at: Date;
+    last_fetch_at: Date;
+    is_anomaly: boolean;
+    is_stale: boolean;
+    trade_url: string | null;
+    token_info_url: string | null;
+    coin_id: string;
+    target_coin_id: string;
+    asdasd: boolean;
```
Comment (janklimo) [inline types/coingecko-api/index.d.ts:353]: 👀

**Review by janklimo — CHANGES_REQUESTED** [review]
One line seems to be a leftover

File: types/coingecko-api/index.d.ts (hunk 1 above)
Comment (JonathanTheZero) [reply types/coingecko-api/index.d.ts]: Oh, you're right, that has been fixed, thanks!

**Review by janklimo — APPROVED** [review]
💯

**Comment by JonathanTheZero** [comment]
Ready to merge

**Pull request merged**